android/src/main/java/com/baseflow/flutter/plugin/geolocator/data/wrapper/MethodResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.baseflow.flutter.plugin.geolocator.data.wrapper;
+
+import io.flutter.plugin.common.MethodChannel;
+
+/**
+ * Instances of this class are invalidated once the wrapped {@link MethodChannel.Result}
+ * instance has been used for communication.
+ */
+public class MethodResponse extends ChannelResponse {
+
+    private final MethodChannel.Result result;
```

Comment:
Rename to match with naming convention:

```suggestion
    private final MethodChannel.Result mResult;
```